src/components/general-page.css

```diff
@@ -201,48 +143,41 @@
 
 .ols-plugin__context-menu {
   max-width: 16rem;
-  padding: var(--pf-v5-global--spacer--md);
+  padding: var(--pf-t--global--spacer--md);
 }
 
 .ols-plugin__context-menu .ols-plugin__context-label {
-  margin-bottom: var(--pf-v5-global--spacer--md);
-}
-
-.ols-plugin__context-menu-heading {
-  color: var(--pf-v5-global--Color--400);
+  margin-bottom: var(--pf-t--global--spacer--md);
 }
 
 .ols-plugin__context-menu-icon--active {
-  color: var(--pf-v5-global--primary-color--100);
+  color: var(--pf-t--global--icon--color--brand--clicked);
 }
 
-.ols-plugin__footer {
-  margin: var(--pf-v5-global--spacer--sm) auto;
+.ols-plugin__footnote {
+  color: var(--pf-v6-c-content--small--Color);
```

Comment:
I wonder if there are version agnostic tokens available for this case

Reply:
👍